Makefile

```
@@ -6,11 +6,13 @@ PIP ?= pip
 VENV = virtualenv
 VENV_ARGS = -p $(PYTHON)
 VENV_DIR = $(CURDIR)/.venv
+VENV_DATA_DIR = $(CURDIR)/data
```

Even if it will be loaded with the virtualenv, I think we can call it `WATSON_DIR` to be more explicit:
```suggestion
WATSON_DIR = $(CURDIR)/data
```